mozilla-central-patches/disable-search-tests.diff

```diff
@@ -0,0 +1,49 @@
+diff --git a/browser/components/search/test/browser.ini b/browser/components/search/test/browser.ini
+--- a/browser/components/search/test/browser.ini
++++ b/browser/components/search/test/browser.ini
+@@ -14,40 +14,44 @@ support-files =
+   tooManyEnginesOffered.html
+   webapi.html
+ 
+ [browser_426329.js]
+ [browser_483086.js]
+ [browser_addEngine.js]
+ [browser_amazon.js]
+ [browser_amazon_behavior.js]
++skip-if = true # mozilla/activity-stream#2339
```

Comment:
8 lines of context seems likely to conflict with any `browser/components/search/test/browser.ini` changes. Just one line of context should be good enough to skip the specific next test.